docs/stdlib/bytes.rst

```diff
@@ -55,16 +57,16 @@ Bytes
         db> select b'Hello,\x20world\x01';
         {b'Hello, world\x01'}
 
-    There are also some :ref:`generic <ref_std_generic>`
-    functions that can operate on bytes:
+    Additionally, :ref:`generic <ref_std_generic>`
+    functions are able to act upon bytes:
 
     .. code-block:: edgeql-repl
 
         db> select contains(b'qwerty', b'42');
         {false}
 
-    It is possible to :eql:op:`cast <cast>` between :eql:type:`bytes` and
-    :eql:type:`json`. Bytes are represented as base64 encoded strings in json.:
+    It is possible to :eql:op:`cast <cast>` between bytes and
```

Comment:
Mirroring the change in casting information for the boolean type:
>    You may :eql:op:`cast <cast>` a JSON value into bytes.

The next sentence feels to me like a note after the example, but I could see an argument for keeping it here too. I've not formatted this as a suggestion so you can decide where you think is the best place for the info about Base64 encoding.